staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -912,6 +912,11 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdown
 	// +optional
 	ShutdownGracePeriodCriticalPods metav1.Duration `json:"shutdownGracePeriodCriticalPods,omitempty"`
+	// PodPriorityShutdownGracePeriods specifies the shutdown grace period for Pods based on their associated priority class value. Requires the GracefulNodeShutdown feature gate to be enabled.
```

Comment:
Use `camelCase` for the field name since this is an external field (see other fields).

Also, add a few sentences describing the behavior of the priority grace period in terms of shutdownw as an admin should understand it.  Specify what happens when this is specified and shutdownGracePeriodCriticalPods is also set.